democracy/migrations/0009_section_type_model.py

```diff
@@ -7,12 +7,48 @@
 from django.db import migrations, models
 import django.db.models.deletion
 import django.utils.timezone
-from democracy.models.initial_data import create_initial_section_types
+
+
+class InitialSectionType:
```

Comment:
Hmm. Will it actually matter if this migration creates the section types with new data?